docs/Urunc-Hypervisors.md

```diff
@@ -0,0 +1,63 @@
+# Installing urunc with all supported hypervisors
+
+In this document, we will go through the installation process of the various hypervisors currently supported by `urunc`.
+
+## urunc with solo5-hvt
+
+First, let's install the apt packages required to build solo5:
+
+```bash
+sudo apt-get install libseccomp-dev pkg-config gcc -y
+```
+
+Next, we can clone, build and install `solo5-hvt`.
+
+```bash
+git clone -b v0.6.9 https://github.com/Solo5/solo5.git
+cd solo5
+./configure.sh  && make -j$(nproc)
+sudo cp tenders/hvt/solo5-hvt /usr/local/bin
+```
+
+Next, we need to configure the [devmapper snapshotter](https://github.com/nubificus/urunc/blob/main/docs/Installation.md#setup-thinpool-devmapper).
+
+Now we can run a test unikernel:
+
+```bash
+sudo nerdctl run --rm -ti --snapshotter devmapper --runtime io.containerd.urunc.v2 harbor.nbfc.io/nubificus/urunc/redis-hvt-rump:latest unikernel
+```
+
+## urunc with qemu
+
+Let's install QEMU:
+
+```bash
+sudo apt-get install qemu-kvm libvirt-daemon-system -y
```

Comment:
I think that `libvirt-daemon-system` is not required to install QEMU.

Reply:
Yes, you are right. I will edit the command.